deploy/cr-minimal.yaml

```diff
@@ -3,7 +3,7 @@ kind: PerconaServerMongoDB
 metadata:
   name: minimal-cluster
 spec:
-  crVersion: 1.18.0
+  crVersion: 1.19.0
```

Comment:
If we are keeping 1.18.0 images in this PR, maybe let's update crVersion in next pr where we will revert images?